app/views/notes/_form.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_with model: [ current_user, current_user.notes.build ] do |form| %>
+  <div id="align-form">
+    <div class="form-group">
+      <h5>Add a Note</h5>
+      <label for="note">Title</label>
+      <%= form.text_field :title, class: 'form-control' %>
+      <label for="note">Note</label>
+      <%= form.text_area :body, class: 'form-control' %>
+    </div>
+    <div class="form-group"> 
+      <%= form.submit 'Submit', class: 'btn btn-success' %>
+    </div>
+  </div>
+<% end %>
+    
```

Comment:
extra line is still there.

Reply:
fixed

Reply:
![Screenshot from 2022-02-22 12-31-33](https://user-images.githubusercontent.com/82970871/155083831-706cc758-622d-4522-8e8d-680efbc874c2.png)